Card.cs

```diff
@@ -0,0 +1,68 @@
+﻿using EnumColors;
+using System;
+
+namespace Task1_GUI
+{
+    internal class Card : IComparable
+    {
+        private int setterNominal;
+        private Colors.SevenColors setterColor;
+        public int Nominal { get; }
+        public Colors.SevenColors Color { get; }
+        public static int NumOfWinner { get; set; }
```

Comment:
Why this property is static?

Reply:
Because it must be common to all objects of the class